docs/sdk_performance.md

```diff
@@ -0,0 +1,57 @@
+# SDK Performance
+
+## SDK Metrics
+
+| Size   | RAM overload at peak | Batch average size | Batch max frequency |
+|--------|----------------------|--------------------|---------------------|
+| 612 KB | 6%                   | 10000 KB           | 1 second            |
+
+### Notes
+
+1. The SDK size was measured by computing the difference between the sizes of our sample application with and without
+the Datadog SDK dependency. Please have in mind that this number is volatile and it takes into 
+account also the SDK transitive dependencies. It can go up or down if your app has or not transitive
+dependencies in common with the SDK. For now the SDK uses the following transitive dependencies: 
+    -   androidx.core:core
+    -   androidx.navigation:navigation-fragment-ktx
+    -   androidx.navigation:navigation-ui-ktx
+    -   androidx.navigation:navigation-runtime-ktx
+    -   androidx.recyclerview:recyclerview
+    -   androidx.work:work-runtime
+    -   com.google.code.gson:gson
+    -   com.lyft.kronos:kronos-android
+    -   com.squareup.okhttp3:okhttp
+    -   org.jetbrains.kotlin:kotlin-stdlib
```

Comment:
Is it with a debug or release build? Even without minification/obfuscation, a release build is usually smaller as it doesn't keep all the debug info.
Also we should add the size of the lib itself without the dependencies, cause some of those are quite heavy, it would show how small our SDK is.

Reply:
it is a release build. We can add the size of the SDK also without the deps of course.